### DISCUSSION TITLE
Forest carbon flux data comment from author

### DISCUSSION THREAD

**Discussion by dagibbs22** [samapriya/awesome-gee-community-datasets#39] [General]
Hello Samapriya,

I am David Gibbs, one of the authors of the [forest carbon flux data ](https://samapriya.github.io/awesome-gee-community-datasets/projects/cflux/) you added to your catalog earlier this month. Thanks very much for adding it. I'm glad you find our data awesome! How did you hear about it?

I noticed a few things about your post. 
1. I wanted to confirm that you did use the 2021 version of the data. It was updated around April 18. 
2.  Assuming that you did use the 2021 version of the data, in a few places it says that users should divide by 20, but they should divide by 21. Our metadata was a little out-of-date and has since been corrected. 
3. The sample code (https://code.earthengine.google.com/d6d6390f0a9eafdaeb9766022b64065e) doesn't load the assets. I tried making my own script to load the assets but still couldn't load them ("Gross removals (no viz): Layer error: Image.load: Image asset 'projects/sat-io/open-datasets/forest_carbon_fluxes/gross_removals' not found."). 
4. Would you be interested in including a link to the Global Forest Watch map viewer where we show the data? The link would be https://gfw.global/3jLklJ9. 

Again, thanks for sharing our data with the world. You can reply here or email me at david.gibbs@wri.org.

Best wishes,

David Gibbs
Global Forest Watch, World Resources Institute

**Comment by samapriya** [comment]
Hey @dagibbs22 
Thank you for reaching out I believe someone from the community reached out trying to access it and that is how I discovered the paper and the datasets. 

* I did use the 2021 version of the data and I have now updated the metadata to divide by 21
* The sample code should now work I had moved the data around so thanks for pointing that out 
* I absolutely should have included the GFW link and you will notice that I have now included that too

Please let me know if there is anything else you would like to be changed/updated. It has been great working with WRI datasets and I can see so many more being ported over. 

Best Wishes,
Sam

**Comment by dagibbs22** [comment]
Hi Sam,

Thanks very much for making those changes so quickly. I hope your GEE assets get lots of traffic.

Best wishes,

David

**Comment by dagibbs22** [comment]
Hi Sam,

There's an updated version of the flux model on GEE through 2022. I hope you can update your GEE assets and the metadata. Data sets are shown at https://code.earthengine.google.com/ae55707e335894d7be515386195390d2. Metadata is in the GFW Open Data Portal (e.g., https://data.globalforestwatch.org/datasets/forest-greenhouse-gas-emissions/explore). Thanks. Please let me know if you have any questions. 

David

**Comment by samapriya** [comment]
This will be updated in next release on Monday thanks for letting me know

**Comment by samapriya** [comment]
This has been updated :) let me know if there are remaining issues

**Comment by dagibbs22** [comment]
Thanks very much. The link https://code.earthengine.google.com/?scriptPath=users/sat-io/awesome-gee-catalog-examples:agriculture-vegetation-forestry/GLOBAL-FOREST-CARBON-FLUXES doesn't seem to be working.

Also, since you don't seem to track when we update our data and I might not remind you in the future, could you please put at the top of the data page (https://gee-community-catalog.org/projects/cflux/) something like the following text: "NOTE: This page may not contain the latest version of the data described below. Please refer to [Global Forest Watch](https://gfw.global/3jLklJ9) for the latest data." That way people know to check the official page for the latest data from the start. Thanks.

**Comment by samapriya** [comment]
sure is there  a direct page where you list the earth engine collections always? I don't see that as a data option

**Comment by dagibbs22** [comment]
The three EE assets aren't all listed together anywhere. Each one is linked on its corresponding GFW Open Data Portal geotif page: https://data.globalforestwatch.org/datasets/gfw::forest-greenhouse-gas-emissions/about, https://data.globalforestwatch.org/datasets/gfw::forest-greenhouse-gas-net-flux/about, https://data.globalforestwatch.org/datasets/gfw::forest-carbon-removals/about

**Reply by samapriya** [reply]
That makes sense I will write something to occasionally check for updates and will also post the check back gfw for latest updates

**Comment by samapriya** [comment]
The datasets have been updated to reflect the April release

**Reply by dagibbs22** [reply]
Thanks very much.

**Comment by dagibbs22** [comment]
Melissa Rose told me you talked with her about our forest carbon flux model assets being in your community catalog. It sounds like you aren't using the latest version, the assets for which you can find at https://code.earthengine.google.com/59ab5af5688f7982ed5f7fa4f56c7c0a?. I thought you were going to write a script that checked for updates; we completed this update almost 6 months ago. Thanks for posting our data.

**Comment by samapriya** [comment]
Thanks for the info and the sample script @dagibbs22 this has been updated now along with documentation on the community catalog. As you can imagine I run this by catalog by myself so it helps to have the community help create update tickets. And yes I was going to write a script that checked for updates but just like a lot of things it's still on the to do list :)

**Reply by dagibbs22** [reply]
Thanks! We'll try to remind you when we do the next update (spring or summer 2026). And thanks again for sharing our data.
